config/initializers/devise.rb

```diff
@@ -0,0 +1,303 @@
+# frozen_string_literal: true
+# rubocop:disable Metrics/LineLength
+
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  config.navigational_formats = []
+  # Ensure you have defined default url options in your environments files.
+  # Here is an example of default_url_options appropriate for a development
+  # environment in config/environments/development.rb:
+
+  # config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+
+  # In production, :host should be set to the actual host of your application.
+
+  # The secret key used by Devise. Devise uses this key to generate random
+  # tokens. Changing this key will render invalid all existing confirmation,
+  # reset password and unlock tokens in the database.  Devise will use the
+  # `secret_key_base` as its `secret_key` by default. You can change it below
+  # and use your own secret key.
+  # config.secret_key = 'a48de4fb5efd22810beefccd09c3ff31fab3a8f6a5cb50cd4120e6ffb134fd65b71091f50bd226eb34e6096dad95aa0a9f11150410f1ee036599d9f9ee589ce3'
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer, note
+  # that it will be overwritten if you use your own mailer class with default
+  # "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```

Comment:
We need to configure this sooner or later. I guess it's out of scope for this PR, but maybe add a note so that we don't forget?